woocommerce/wc-admin
# Remove items from search input with backspace key

Fixes #1352 

The title is pretty self explanatory on this one! 😅

### Screenshots

![backspace_test](https://user-images.githubusercontent.com/1177726/52574754-d9cfab00-2e14-11e9-9887-a1ff4b8f1eea.gif)

### Detailed test instructions:

* Go to the _Customers_ report and select _Advanced Filters_ under the _Show_ dropdown.
* Add a filter of type _Name_.
* Write a couple of names.
* Press the backspace.
* It should remove the last item
